server/model/repo.go

```diff
@@ -26,6 +26,7 @@ import (
 type Repo struct {
 	ID                           int64          `json:"id,omitempty"                    xorm:"pk autoincr 'repo_id'"`
 	UserID                       int64          `json:"-"                               xorm:"repo_user_id"`
+	RemoteID                     string         `json:"-"                               xorm:"UNIQUE 'remote_id'"`
```

Comment:
It would be preferred to also add migration that adds this column using Sync2

Reply:
I know but from experience with Gitea if this is not done as migration and this field will be used in future migrations it can be problem that migration will fail as there will be no such column in table and thus will create problems if upgrades are not made regularly